Model/Adminhtml/Source/Method.php

```diff
@@ -31,7 +31,7 @@ public function toOptionArray()
         if (!$this->options) {
             $this->options = [
                 [
-                    'value' => '',
+                    'value' => 'payment',
```

Comment:
Please double check this change. It was required to make the `<depends>` field work as it doesn't work with empty values. All the uses i could find are ifs on this way:
```
if ($method == 'order') {
  ...
} else {
   // Payment API
}
```

So i think this is a safe change.